src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -887,5 +887,196 @@ public static void CheckLegacyNullRowVersionIsEmptyArray()
qiRgrLtRuQyg5CEBQCVUbymLI655iIA4UT53t4bIvpLe8mDOTO7Oq1q4FGVBFxEDQvmGteEXi02f41mmY3F8tLGV6rZ7AXlDjhp9yjpWwUOnot2VmOefvN0dkc1JZ9bp77FND4ho4MqJUvleVJ7En4TLEm7L31Gj8DuFzLTxuhblVJl7nNJO6KjjNDQEIVVh2uaoyOLDABnz1xLad4OQaeIK5jwWqTECaBioLZJdtiF7QdBemm0vwo2CmhCqhpiQFNbQwZWr9eTtP2BmtP7HhPjMj9WuIJs9vu0EChnHgYF15oAIFg7pFMTOrm42x0HWi0B8pM2MuqjgvZs9ziC2ZnUYNTk825f3dtDEy0E9pacGP6HiFcGnfvwxHkZ9yyaNApFL3AAAaDOTytgqDfWdA2CE7qi0IfY5EC8FQz9LIPCv0Md7IYSqob5S9Cz1m0j6NZ6kHMhxCdIxbo2N5e2R641ztSgYdDtwKw0oQblfNX82uw5NajqvNldPlEGBH0SHE37hPGPQMKCv4vFrJUtMRhjbStnWVJd5I5qQXWul5zO6NCumYcI5Zu1L0bO9qxpYYDAzfs31KzeOti98UCGNSkMlSNxOkRJ8groQSGaQPLNyyngCrKNqrj6Y3K67nNgM80kyJzyFEdBfhrTXiNGeh9msosPIfkmPANucDxPrPqpIDbPuCA8dd7rNDv31XnmLLz9plftCfPfX0oXhCEvEFS6luhA9qcBHIkVQDU0DeQH1o0Uvg7dC0ksuDavDCXTS1hdo1AmjV8cjW6PMlFtxKA15WsNTcV2KyHiBfMNA25s5OTQEd2QkXuc18ld6IJASg7Bjy5DAg1lGplwjfKhGYKSQl5NnDzupAfxr6fKWzHSVoRkoPUPQ1XGzTBbYo49XbFP9K8qXrZFD9AZBTb8ArtmnyWaIwqjPsJYgKFsvwiFD7wZ9969ap6jIqgyGktPw1uqAIgFKBf19b8YHjLa7ZcY8LQcWazg49Xxih9LUkKXY78S72gQUnljgtzMOFrHlQjL17n6cYs6mAR86TsqMpnGl6VA2WIC7uDoB86dGiUOuZujSMoEibAVAciN15w5z0qKQ9qsCvmtCdDAUGHTV1ks9bQR7RnjpxPRhhvIIshRMy7aEUaNnTRlfa8D1EdsCBjrXJqJnimh29dVqVhfZLWiUblmcCaSwoif2eyUSLkdPUMKE1Xf1LW0FqNib0ZnJUHEMxbzpC6afacayA180TMVeOsFB5MbJf5ma5EFW18q181YNa17nLVH6sUQErhbaKvlLLS4tkme4Pp2qScIsJHlrfRx9ytThxLuxlSQ10E3meBNdspNsu1MjnwaEsvYtdA3tUFMHKR7nv9hWCS6l8k8WadGpZPPwLXcQiBlYrUvjigY7UZn2Avv5hADmmTEWv9XjZoG4VvhniXjDmzSagdo3B4bs71jU6P7nkxzpgZJS0L7cJW9jUkwf18LzIr1T3jCQfrvbeqTCliXKekVEbzVYaCKpsq9kbk6tPPfuhDPkc3wayhqj2cD2OvrGpFAgrwJpdzOaHwsayUbJCwaEl94k9koX38yPeoFUo1Bm3cQQLGmN2MpdduQUX7osTfrOUErMzcm3QQKHM3kCYFHx6hEFdb2LGhebvrFnyAgcgICkDhfb8Ya2oWHxNZ2AV8nUZpx4mYIwlTU7MDWAK0oGkxaJ1tCHRIJSPcQf2qlf0AGjtY1QxJcR97rMhiU6rhgDLgLeM9NgoY7jRzPlApNKCc3eRSxxBKB5NeBnPVPeP9VArPmGfcJYltWJCODwqpFRFAFuqrBD247YlBiTAXXn9OfWA5nfVHzAnREFf4lkxQmX7XHbBiD8hsjm3wHDsLLUsR9II9kDGUJSDp5RAW6gfXuk0K8Tf20I0lDbWF7w9fGJkPKtRdgoOJsPN1El0i3CWDkW5FxGwj4kRsLh4o02JKM371NAg3SwIRYFd8Sl15WN5yk6ZB9q3uheNGinAehq2CORuYlWnPjLBR1gI8ons9cwQtwWbZUJ1BkoG0k517gtsk0cbfB0OpyliVOQGWH0XX2Mwj5jymwuNlqjNudb9HYE1XmnyCTpDgKTY0NkDeP7nRidmUlv8UyoDD7Z2L7ZlmJ0Oz5yQx8ccUV9dJO0L8dT08VOUIs5TQ0TWSYkd5hYSaAGvgYXykZLSYNQs4aaMO1wa7LbK6OTjIJnj50o3JRCudq86s08SQ4fwSJcRO9oq2UWtdeZX62z10OdkrVrbRIdTyZlEQSVbTEb77qipRmfORcvlIaz7WGgWBUz7lpUdmOQzfgmEpiNUJiW5IZJTdYmRNYdEM92tLLdYPhiMsjrCqJ73Vd80RoOnrnUwrlNbhBRydHwCjJsYcHbuFywvay5VyqYAJ7cvn1VUGARXxO5yHDO5XJeb11BYQ3NiLcLpNyrU0IdEcffqTFtBq4S0e02aoVUEJAHWFgsGyMJ9MkdnsY11X18YRLqe0TCN0sRMeXRjQaLL3mY6Qf12jqj6L645DVUXGoRkadj6EwJRouxkQiWP7crgHknwyywSSBLKgJsl6kDEsYCSjQP9Q2LSDdWnIbW1qIc4ksKn5TXXB3MgQ4i251utgQf665Plxg33VJpRvP1ueEGLudi2VOC8YTMDULevIXYLwzfIUA53EHPnjlzfIS8v6BzBL2DxoIBWns0J5Ci9rArjPWB3h6UkeAeqXLJsPrfKqesYL5ZJAJPopIXdzRuqseqie1akOWQDcOxxDQsvEWVBfjuZpvaectHK7HlxzgeXP14Aa2tqyUfVpM5meDtcGcPSd1SfNf0RGvD16rgiq225T7yx4c0uBJTRQekF2CXvGUDTDeLxR08GKscXlgBCZOgeOjd4kX3UGXSY7T0KPpKNgLLjwLkPQBqXrJcThYDimnm1zn73HfoPO7qsPqXmB8S380lMQqrXOLYizumMutVQX1D2FWLNLkYlKpUQb8jkT0QX5AfcNPFFKOefyZltq7JCvf4gh7oPTMKe8VVPoGA
```

Comment:
Why not randomize and generate instead?

Reply:
Because while debugging it remembering the substrings and finding them in the code was important. As I noted in the original post the lengths of all the fields are important and it took a couple of hours to get a stable reproduction from the original reporting user that worked for me. The more changes I make the more likely it is to disrupt and make the test useless.